pwnlib/update.py

```diff
@@ -191,10 +191,13 @@ def perform_check(prerelease=current_version.is_prerelease):
     return command
 
 def check_automatically():
+    xdg_config_home = os.environ.get('XDG_CONFIG_HOME') or "~/.config"
```

Comment:
I would recommend loading BOTH files, not one vs. the other.

Reply:
All programs I've ever seen use the same logic I suggested, e.g. 

https://github.com/git/git/blob/e6362826a0409539642a5738db61827e5978e2e4/Documentation/git-config.txt#L293-L294

https://github.com/htop-dev/htop/blob/12c2337939cbdee21c437c28646f128667a162de/Settings.c#L392-L403

Also, the default value for `$XDG_CONFIG_HOME` is defined to be `~/.config`. In fact, I've never seen a system with explicitly defined `$XDG_CONFIG_HOME` environment variable yet.

Reply:
pwntools already does is like this for `$XDG_CACHE_HOME`, see

https://github.com/Gallopsled/pwntools/blob/082de2cc7e6e1dce00d00851fae14741038778f0/pwnlib/context/__init__.py#L1236-L1237